packages/combo-box/src/vaadin-combo-box-scroller.js

```diff
@@ -250,8 +250,8 @@ export class ComboBoxScroller extends PolymerElement {
   }
 
   /** @private */
-  __focusedIndexChanged(index) {
-    if (this.__virtualizer && index >= 0) {
+  __focusedIndexChanged(index, oldIndex) {
+    if (this.__virtualizer && index !== oldIndex) {
```

Comment:
Would it make sense to move `index >= 0` check before calling `this.scrollIntoView(index)`?

Reply:
This is good point, done.